app/models/message.rb

```diff
@@ -0,0 +1,18 @@
+require 'data_mapper'
+require 'dm-postgres-adapter'
+
+class Message
+
+  include DataMapper::Resource
+
+  property :id, Serial
+  property :peep, Text
+  property :name, String
+  property :username, String
+  property :posted_at, Text
+
+end
+
+DataMapper.setup(:default, ENV['DATABASE_URL'] || "postgres://localhost/chitter_#{ENV['RACK_ENV']}")
```

Comment:
Metrics/LineLength: Line is too long. [100/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)